### DISCUSSION TITLE
來蓋個笑話樓

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1800] [General]
留言區留下你最近聽到的笑話
我先來:
1.胡迪死了要播甚麼歌
.
.
.
.
盛夏光年

2.為甚麼奴隸脾氣都很好
.
.
.
.
因為他們簽了賣身契

3.從前有隻鵪鶉
別的鵪鶉都早上下蛋
只有他晚上下蛋
所以別人都叫他
晚鵪鶉蛋

**Comment by davidteng00000** [comment]
我出門前對桌上的隱形眼鏡說：
.
.
我根本沒把妳放在眼裡

**Comment by 111426027** [comment]
巧克力和草莓打架，為什麼是巧克力贏?
.
.
.
因為巧克力棒。(雖然我覺得很爛哈哈)

**Comment by davidteng00000** [comment]
一個真正好的女孩
她們不需要你幫她們付帳
不需要你長得很帥
不需要你多金
不需要你每天接送
不需要你送昂貴的禮物
不需要你獻殷勤
.
.
.
.
也不需要你

**Comment by eric202501** [comment]
媽媽: 小陳! 不可以用手指月亮。
小東: 蛤?

**Comment by 12Jesse24** [comment]
那我也分享一下前幾天聽到的笑話

1.有 一天DNA想要養狗
.
.
.
.
.
但我們都叫他去氧(養)核醣核酸


2.民進黨為什麼打籃球都輸
.
.
.
.
.
因為它們都不投籃(藍)


3.為什麼8+9只知道25個英文字母
.
.
.
.
.
因為K被吸掉了(這個好刻版印象)

**Comment by Patsy17** [comment]
我：醫生，我手術後要多久才能拉小提琴？
醫生：一個月。
我：謝謝你，我本來不會拉的！

**Comment by Tomken123** [comment]
我活得像個笑話

**Comment by mrweixd** [comment]
打電話給一個coser

「喂？」

好
他現在變-siner了

**Comment by alvin119** [comment]
有一次生物考試。其中有一題是要看鳥的腳的圖片，寫出鳥的名字。
某考生因為看不懂，所以很生氣的把考卷撕爛走出考場。
監考老師看到了很生氣的說：同學你回來！你是哪班的！叫什麼名字！
考生彎下腰，捲起褲管，指著自己的腿很大聲的跟老師說：你猜啊！你猜啊！

**Comment by chiuwangwang** [comment]
阿杏摔倒變什麼.............
...................
...................
...................
...................
...................
>
阿呆





對不起，我先道歉，這真的太爛了，但不能只有我知道，分享給大家。

**Comment by William08290829** [comment]
有一題數學題如下:

爸+媽=30歲
爸-媽=18歲

試問
，
，
，
，
，
，
，
，
，
爸爸要關幾年

**Comment by jeff110303549** [comment]
每種金屬都有自己獨特的含義
金是高貴
銀是優雅
鋁是軒
鎂是咖啡
氡是買駿馬 錫是買鞍韉
鉀是聽Bieber
鈾舔嘴唇 鈣是英雄
銫是25度C
鈦是奶茶 釩是林
鋅是紀福音戰士
鍺是什麼 鈉是什麼

**Comment by jeff110303549** [comment]
能蟲，能蟲屬於昆蟲綱鞘翅目，雜食非常凶殘，體內經常有很多病毒。在唐朝就已經‌‌‌‌​​​​‌​‌‌‌‌‌‌​‌‌出現，被劍客李白一劍殺死。有文獻記載：要是能蟲來，我要選李白。

**Comment by thomasxshen** [comment]
Ni是鎳
Cr是鉻
NiCr是甚麼?

**Comment by EPHMR0524** [comment]
你們知道校門口的警衛在笑什麼嗎？
.
他在校門口

一隻兔子叫做rabbit 八隻兔子叫做rabyte

寫程式時code寫得太亂了找不到array 的index變數是哪一個
不知索引（index）

這是我所有的笑話了，希望有好笑到

**Comment by thomasxshen** [comment]
Q:一隻小貓被一群野狗追殺到死巷時，小貓忽然轉身對這群野狗說了七個字，只見那群野狗嚇的落荒而逃，你知道那七個字嗎？
A:霹靂-霹靂-霹靂貓

**Comment by thomasxshen** [comment]
Q:一個池塘裡有五隻魚，忍者對那五隻魚分別射出五隻飛鏢，請問忍者射到了幾隻？
A:兩隻，因為三隻魚閃鏢（三支雨傘標）

**Comment by thomasxshen** [comment]
Q:有四個人分別叫帥酷哥、酷酷哥、帥帥哥、酷帥哥、請問誰的老婆最容易生病？
A:酷酷哥因為他的老婆叫酷酷嫂

**Comment by jeff110303549** [comment]
剛走在空無一人的路上
突然聽到有人在念3.1415926......
我就嚇得趕快跑，回家我媽問我怎麼氣喘吁吁的
.
我說我遇到π郎

**Comment by thomasxshen** [comment]
Q:灰熊、白熊、黑熊，誰厲害大家都知道是灰熊厲害嗎？但是灰熊和老虎時，誰厲害勒？
A:呵呵..當然是虎囉(好厲害台語)

**Comment by 110101501** [comment]
我出門前對桌上的隱形眼鏡說






我根本不把你放在眼裡
